### PR TITLE
Overhaul of entire site.

### PR DESCRIPTION
**Pull request by YodaForce157**
Added a slideshow of images, contributors, downloads page etc.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <div class="navbar">
+            <div class="navbar-left">
+                <img src="images/logo.png" alt="Railroad IDE Logo">
+                <h1>Railroad</h1>
+            </div>
+            <div class="navbar-right">
+                <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="downloads.html">Downloads</a>
+                <a href="discord.html">Discord</a>
```
Comment (DaRealTurtyWurty) [inline about.html:30]:
```suggestion
                <a href="https://discord.gg/BAYB3A38wn">Discord</a>
```

File: downloads.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <div class="navbar">
+        <div class="navbar-left">
+            <img src="images/logo.png" alt="Railroad IDE Logo">
+            <h1 id="logotext">Railroad</h1>
+        </div>
+        <div class="navbar-right">
+            <a href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="downloads.html">Downloads</a>
+            <a href="discord.html">Discord</a>
```
Comment (DaRealTurtyWurty) [inline downloads.html:27]:
```suggestion
            <a href="https://discord.gg/BAYB3A38wn">Discord</a>
```

File: index.html [hunk 3]
```diff
@@ -1,22 +1,53 @@
...
+  <div class="navbar">
+    <div class="navbar-left">
+      <img src="images/logo.png" alt="Railroad IDE Logo">
+      <h1>Railroad</h1>
+    </div>
+    <div class="navbar-right">
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
+      <a href="downloads.html">Downloads</a>
+      <a href="https://discord.gg/xGUNJKGGnW">Discord</a>
```
Comment (DaRealTurtyWurty) [inline index.html:28]:
```suggestion
      <a href="https://discord.gg/BAYB3A38wn">Discord</a>
```

**Review by DaRealTurtyWurty — CHANGES_REQUESTED** [review]

**Comment by YodaForce157** [comment]
All sorted :)

**Review by DaRealTurtyWurty — APPROVED** [review]

**Pull request merged**